### PR TITLE
[stable/elasticsearch] updated es to 7.1.1

### PR DESCRIPTION
**Pull request by monotek**
Signed-off-by: André Bauer <monotek23@gmail.com>

#### What this PR does / why we need it:
* updates es to 7.1.1

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/elasticsearch/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: elasticsearch
 home: https://www.elastic.co/products/elasticsearch
-version: 1.28.5
-appVersion: 6.7.0
+version: 1.29.0
+appVersion: 7.7.1
```
Comment (McQueen2063) [inline stable/elasticsearch/Chart.yaml:5]: shouldn't that be 7.1.1 instead of 7.7.1 ? :)

**Comment by monotek** [comment]
Es chart will be deprecated in favour of the official elastic charts. Closing...

**Pull request closed without merging**